runners/flink/build.gradle

```diff
@@ -82,6 +83,7 @@ dependencies {
   validatesRunner project(path: ":beam-sdks-java-core", configuration: "shadowTest")
   validatesRunner project(path: ":beam-runners-core-java", configuration: "shadowTest")
   validatesRunner project(path: project.path, configuration: "shadow")
+  testCompile project(":beam-sdks-java-harness")
```

Comment:
`testCompile -> shadowTest` since we do not plan to shade this away in the shaded test jar.

Reply:
thanks!